dotnet/src/Connectors/Connectors.Amazon/Bedrock/Extensions/BedrockKernelBuilderExtensions.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using Amazon.BedrockRuntime;
+using Amazon.Runtime;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.Amazon;
+using Microsoft.SemanticKernel.Connectors.Amazon.Core;
+using Microsoft.SemanticKernel.TextGeneration;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Extensions for adding Bedrock services to the application.
+/// </summary>
+public static class BedrockKernelBuilderExtensions
```

Comment:
It looks like methods in `BedrockKernelBuilderExtensions` and `BedrockServiceCollectionExtensions` are the same. In order to re-use them, `BedrockKernelBuilderExtensionds` could have a code to call related methods from `BedrockServiceCollectionExtensions` via `builder.Services.AddBedrockChatCompletionService` etc.